src/blockchains/eth/eth-watch.ts

```diff
@@ -158,7 +157,12 @@ export const start: IBlockchainReader = async listener => {
   )
 
   let [db, conn] = await connectToRethink(options)
-  const fromBlock = await getLastBlock(db, conn, options.rethinkTable)
+  console.log(`[ETH] Getting last block from RethinkDB`)
+  try {
+    const fromBlock = await getLastBlock(db, conn, options.rethinkTable)
+  } catch (err) {
+    const fromBlock = process.env.DUCOR_ETH_FROM_BLOCK! || 0;
```

Comment:
Priority:
1. Get last block from DB.
2. If not found or 0, get last block from environment variable.
3. If not found, use `0` as default.